app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class UserMailer < ActionMailer::Base
+  layout 'mailer'
+  default from:  ENV['USER_MAILER_DEFAULT']
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.